src/main/java/duke/exceptions/InvalidCmdValueException.java

```diff
@@ -0,0 +1,8 @@
+package duke.exceptions;
+
+public class InvalidCmdValueException extends Exception {
```

Comment:
I like the level of abstraction for exceptions, but maybe spelling out the full word would be better for naming?

Reply:
Renamed with full spelling.